src/main/java/org/aksw/iguana/cc/controller/MainController.java

```diff
@@ -74,4 +89,14 @@ public static void main(String[] argc) {
         System.exit(0);
     }
 
+    private static void outputVersion() throws IOException {
+        ClassLoader classloader = MainController.class.getClassLoader();
+        String properties = new String(Objects.requireNonNull(classloader.getResourceAsStream("version.properties")).readAllBytes());
+        String[] lines = properties.split("\\n");
+        String projectVersion = lines[0].split("=")[1];
+        String ontologyVersion = lines[1].split("=")[1];
+        System.out.println("IGUANA version: " + projectVersion);
```

Comment:
Please make the output match the following pattern: `IGUANA version 4.1.0 (result ontology version 4.0.0)`